test/runtests.jl

```diff
@@ -4,6 +4,6 @@ using Test
 include("test_without_key.jl")
 
 # Normal usage - API key must be present in ENV
-if haskey(ENV, "FRED_API_KEY")
+if fred_key() !== nothing
```

Comment:
see above, is there another way to make this check without comparing against a sentinel value of `nothing`?
- can create a new method to check if the key can be loaded
- can try to load the key here directly and catch errors

Reply:
I was trying to follow this pattern: https://docs.julialang.org/en/v1/manual/faq/#faq-nothing-1 and https://docs.julialang.org/en/v1/manual/types/#Type-Unions-1 . 

If you would still like me to change it, I would rather not try catch. If you would like, I can make a boolean function that checks env and the file location, and then make a load that gets the value.

Reply:
Reread your comment above. I'll put a boolean check function, and a throw inside the load function.